website/docs/reference/events-logging.md

```diff
@@ -119,12 +117,10 @@ Many events are fired while compiling or running a specific DAG node (model, see
 }
 ```
 
-</VersionBlock>
-
 ## Python interface
 
 :::warning
-The `EVENT_HISTORY` object has been deprecated and removed in dbt Core v1.4+
+The `EVENT_HISTORY` object has been deprecated.
```

Comment:
does it make sense to remove this callout? it reads like a release note entry.

Reply:
Good call! I'll remove it